flink-test-utils-parent/flink-connector-testing/README.md

```diff
@@ -0,0 +1,44 @@
+# Flink Connector Testing Framework
```

Comment:
We should eventually have add this (and expand) to docs. However, I prefer to first use it internally for a while until we advertise it. WDYT?

Reply:
Agree. We can keep it internal for now and publish it when stable in the future.